src/components/atoms/MenuLink/index.tsx

```diff
@@ -3,15 +3,16 @@ import { StyleNavMenuLink, StyleLink, StyleLinkName } from "./styles"
 
 
 interface MenuLinkInterface {
-    linkName: string
+    linkName: string;
+    slug: string;
 }
 
 const MenuLink: React.FC<MenuLinkInterface> = (
```

Comment:
You can also do: 

```
const MenuLink: React.FC<MenuLinkInterface> = (
    props: MenuLinkInterface
) =>  (
        <StyleNavMenuLink>
            <StyleLink to={`./${props.linkName}`}><StyleLinkName>{props.linkName}</StyleLinkName></StyleLink>
            <StyleLink to={`/${props.slug}`}><StyleLinkName>{props.linkName}</StyleLinkName></StyleLink>
        </StyleNavMenuLink>
    )
```

I removed the `return`

Reply:
Did you mean to put two StyleLink in there?